components/permissions/android/BUILD.gn

```diff
@@ -0,0 +1,5 @@
+import("//build/config/android/rules.gni")
```

Comment:
nit: Brave copyright header

Reply:
are we adding it to gn/gni? all gn files I saw didn't have one.

Reply:
yes, but it is not strict checked because lint script doesn't check this.

These have the header:
https://github.com/brave/brave-core/blob/master/base/BUILD.gn
https://github.com/brave/brave-core/blob/master/android/brave_java_resources.gni

And you are right, lots doesn't. So I don't insist, but it would nice to have.
One day I believe it will be forced by lint CI check.